scanners/nmap/values.yaml

```diff
@@ -11,10 +11,14 @@ parserImage:
   # @default -- defaults to the charts appVersion
   tag: null
 
-scannerJob:
-  # scannerJob.ttlSecondsAfterFinished -- Defines how long the scanner job after finishing will be available (see: https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/)
+parseJob:
+  # parseJob.ttlSecondsAfterFinished -- seconds after which the kubernetes job for the parser will be deleted. Requires the Kubernetes TTLAfterFinished controller: https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/
   ttlSecondsAfterFinished: null
 
+scannerJob:
+# scannerJob.ttlSecondsAfterFinished -- seconds after which the kubernetes job for the scanner will be deleted. Requires the Kubernetes TTLAfterFinished controller: https://kubernetes.io/docs/concepts/workloads/controllers/ttlafterfinished/
+  ttlSecondsAfterFinished: 0
```

Comment:
Why is this 0 instead of null as default 🤔

Reply:
Oh great find, was using nmap as my local test case and apparently screwed up changing it back to default 🤦‍♂️